### PR TITLE
Move exp_queue from repository.py to utils.py 

### LINKED ISSUE #971: Move exp_queue from repository.py to utils.py
Since it is now being used in other places too and it is not directly related to repository.py, we should move exp_queue to utils.py.

### PR DESCRIPTION
**Pull request by horacix**
Fixes #971

### CODE REVIEW

File: tests/test_repository.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import responses
 from dateutil.relativedelta import relativedelta
 
-from bugbug import repository
+from bugbug import repository, utils
```
Comment (marco-c) [inline tests/test_repository.py]: Need to move the ExpQueue tests to test_utils.py, as test_repository.py is only meant to test repository.py functions.

**Comment by marco-c** [comment]
Looks good, just one minor thing to fix.

File: tests/test_repository.py (hunk 1 above)
Comment (horacix) [reply tests/test_repository.py]: Of course. I'll push an update tomorrow.

Comment (horacix) [reply tests/test_repository.py]: done

**Pull request merged**